### ISSUE TITLE
ERA001 conflicts with ruff: ignore supression

### ISSUE THREAD

**Issue by 93578237** [astral-sh/ruff#25535]
```python
# ruff: ignore[ARG001]
def foo(bar, baz):
    return bar * 2
```
Run `ruff check --isolated --select ARG,ERA --preview t2.py`:
```
error[[ERA001](https://docs.astral.sh/ruff/rules/commented-out-code)]: Found commented-out code
 --> t2.py:1:1
  |
1 | # ruff: ignore[ARG001]
  | ^^^^^^^^^^^^^^^^^^^^^^
2 | def foo(bar, baz):
3 |     return bar * 2
  |
help: Remove commented-out code

Found 1 error.
```

**Comment by ntBre** [comment]
Thanks for the report! Yep, this reproduces in the [playground](https://play.ruff.rs/d6cd0f15-bbbe-494c-b154-ac3c2f5c0b56) too with your example. Unfortunately `ruff: ignore[ARG001]` is a valid type annotation, so I guess we'll need to exclude it directly in the allowlist with the other suppression comments:

https://github.com/astral-sh/ruff/blob/06e77271bb4522ded6feb45269e86b4a5e8d454c/crates/ruff_linter/src/rules/eradicate/detection.rs#L17
